privaterelay/migrations/0009_remove_duplicate_index.py

```diff
@@ -0,0 +1,49 @@
+# Generated by Django 4.2.7 on 2023-11-13 21:03
+# This index was added in 0006_add_email_indexes.py
+# The same index was added by django-allauth 0.56.0
```

Comment:
concern (blocking): Do we know that `python manage.py migrate` will add the index in the `django-allauth` migration before this `privaterelay` migration removes it? What happens if there's temporarily no index on the `account_emailaddress.email` field?

Reply:
E.g., I checked out the branch and I pip installed the new requirements. Then I ran `python manage.py migrate` but I didn't see the new migration run?
```
 lcrouch  (e) relay-python-3.10  …  fx-private-relay   e496846b ⚑ 17  python manage.py migrate
Not using django_ftl autoreloader because pyinotify is not installed. Set `Bundle.auto_reload` or `AUTO_RELOAD_BUNDLES` to `False` to disable this warning.
System check identified some issues:

WARNINGS:
?: (dockerflow.health.W001) Unapplied migration privaterelay.0008_socialaccount_provider_add_default
?: (dockerflow.health.W001) Unapplied migration privaterelay.0009_remove_duplicate_index
Operations to perform:
  Apply all migrations: account, admin, auth, authtoken, contenttypes, emails, phones, privaterelay, sessions, silk, sites, socialaccount, waffle
Running migrations:
  Applying privaterelay.0008_socialaccount_provider_add_default... OK
  Applying privaterelay.0009_remove_duplicate_index... OK
 lcrouch  (e) relay-python-3.10  …  fx-private-relay   e496846b ⚑ 17  pip freeze | grep allauth
django-allauth==0.60.0
```

I *DO* see index though:
```
    "account_emailaddress_upper" btree (upper(email::text))
```

Reply:
Yup, looks like I had already run the migrations in the past. All good then.